public/plugin.ts

```diff
@@ -78,6 +78,17 @@ export class OpendistroSecurityPlugin
       },
     });
 
+    core.application.register({
+      id: 'multitenancy',
+      title: 'Security',
+      appRoute: SELECT_TENANT_PAGE_URI,
```

Comment:
Minor: better to have same style as line 72

Reply:
I think Line#72 is in redundant since it is directly the variable. I will unify that.